packages/ladle/lib/shared/types.ts

```diff
@@ -130,8 +130,8 @@ export type StoryProps = {
   config: Config;
 };
 
-export type StoryDecorator<P = {}> = (
-  Story: React.FC<P>,
+export type StoryDecorator = (
```

Comment:
@tajo Technically it's a breaking change, if someone just updated ladle to 2.10.0 and started using `StoryDecorator<Props>` then they will have to roll that back. I'll leave it to your consideration. If you want to leave an escape hatch for this scenario at the expense of code clarity, roll 802e30b back.